webplugin/template/chat.html

```diff
@@ -21,6 +21,7 @@
         }
         let params = new URLSearchParams(window.location.search);
         let appId = params.get('appId');
+        let noFocus = params.get('noFocused');
```

Comment:
no able to understand why you want to always use 2 different names sometimes inputFocussed and focussed,
 now noFocus and noFocussed
why not use just one name consistently everywhere?

Reply:
Made the changes and tested, Now using the name noFocus everywhere